k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.java

```diff
@@ -241,7 +241,7 @@ public void addBccAddresses(Address... bccRecipients) {
             boolean alreadyVisible = recipientMvpView.isBccVisible();
             boolean singleBccRecipientFromAccount =
                     bccRecipients.length == 1 && bccRecipients[0].toString().equals(bccAddress);
-            recipientMvpView.setBccVisibility(alreadyVisible || singleBccRecipientFromAccount);
+            recipientMvpView.setBccVisibility(alreadyVisible || !singleBccRecipientFromAccount);
```

Comment:
yup 
the bcc field is supposed to be hidden when the only address present is the default one; but instead it was hidden when it had addresses other than the default bcc address